warehouse/query-core/src/main/java/datawave/query/jexl/JexlNodeFactory.java

```diff
@@ -301,17 +302,26 @@ public static JexlNode createNodeTreeFromPairs(ContainerType containerType, Jexl
             JexlNode rightNode = pair.get(1);
             QueryPropertyMarker.Instance leftType = QueryPropertyMarker.findInstance(leftNode);
             QueryPropertyMarker.Instance rightType = QueryPropertyMarker.findInstance(rightNode);
+            boolean strict = false;
             boolean lenient = false;
-            if (leftType.isType(LenientExpression.class)) {
+            if (leftType.isType(StrictExpression.class)) {
+                strict = true;
+                leftNode = leftType.getSource();
+            } else if (leftType.isType(LenientExpression.class)) {
                 lenient = true;
                 leftNode = leftType.getSource();
             }
-            if (rightType.isType(LenientExpression.class)) {
+            if (rightType.isType(StrictExpression.class)) {
+                strict = true;
+                rightNode = rightType.getSource();
+            } else if (rightType.isType(LenientExpression.class)) {
                 lenient = true;
                 rightNode = rightType.getSource();
             }
```

Comment:
These lines are related to the test that you said was a contrived example.  Curious if we need this if we don't want to support that case.

Reply:
hmm.... let me reevaluate this.

Reply:
removed.